gui/src/pages/AddNewModel/ConfigureProvider.tsx

```diff
@@ -118,6 +121,55 @@ function ConfigureProvider() {
       .some((d) => !formMethods.watch(d.key));
   }, [modelInfo, formMethods]);
 
+  const handleOpenRouterSubmit = handleSubmit((data) => {
+    const { apiKey, model } = data;
+
+    if (!apiKey || !model) {
```

Comment:
you can also leverage `vscode.window.showInformationMessage` to show an error message to the user if they attempt to submit without apiKey, rather than disabling the button

Reply:
Okay. Is this the desired behavior?

Reply:
According to the error code I received, vscode.window.showIngormationMessage cannot be used directly here in the GUI webviews.

Reply:
@jpan8866 Added error handling to be viewed by the user and removed the disabling of the button.

Reply:
Yeah if you wanted to implement this, you would have to pass a message to vscode extension from the webview using webtoide protocol, and then activate the message there

Reply:
Interesting.  I will research that.  Do you still want me to implement that? Or is my current implementation okay?